src/rgw/rgw_rados.cc

```diff
@@ -10818,8 +10818,14 @@ int RGWRados::omap_del(rgw_obj& obj, const std::string& key)
 
 int RGWRados::update_containers_stats(map<string, RGWBucketEnt>& m)
 {
+  uint64_t total_count;
+  size_t total_size;
+  size_t total_size_rounded;
   map<string, RGWBucketEnt>::iterator iter;
   for (iter = m.begin(); iter != m.end(); ++iter) {
+    total_count = 0;
+    total_size = 0;
+    total_size_rounded = 0;
```

Comment:
could you change these to `ent.total_count = 0;` etc. so we don't need the local variables?